Source/WebCore/inspector/InspectorCanvas.h

```diff
@@ -130,7 +130,7 @@ class InspectorCanvas final : public RefCounted<InspectorCanvas> {
 
     String m_identifier;
 
-    CanvasRenderingContext& m_context;
+    WeakPtr<CanvasRenderingContext> m_context;
```

Comment:
It bugs me that we're using a WeakPtr but keep assuming that it is non-null. Why didn't you use a CheckedRef<> here?